modules/models/query_builder.php

```diff
@@ -69,6 +69,103 @@ public function order(array $data): QueryBuilder
         return $this;
     }
 
+    public function where_raw_sql(string $raw_sql) {
+        $sql = "SELECT {$this->table_name()}.* FROM {$this->table_name()}";
+
```

Comment:
pretty much a copy and paste of the `where` method to allow for querying with raw sql, it would take too much effort to make it possible to query with bit operators soooo yeah